virtool/administrators/data.py

```diff
@@ -177,3 +177,12 @@ async def delete(self, user_id: str):
             return
 
         raise ResourceNotFoundError()
+
+    async def promote_administrators(self):
+        async with self._db.create_session() as session:
+            async for user in self._db.users.find(
+                {"administrator": True}, session=session
+            ):
```

Comment:
You can use `users.distinct("_id", {"administrator": True})` here to be more efficient.